lib/node_modules/@stdlib/utils/group-in/docs/types/index.d.ts

```diff
@@ -181,8 +182,7 @@ declare function groupIn( obj: any, indicator: Indicator ): any;
 * var out = groupIn( obj, opts, indicator );
 * // e.g., returns { 'b': [ [ 'a', 'beep' ], [ 'b', 'boop' ], [ 'd', 'bar' ] ], 'f': [ [ 'c', 'foo' ] ] }
 */
-declare function groupIn( obj: any, options: Options, indicator: Indicator ): any;
-
+declare function groupIn<T, K>(obj: T, options: Options, indicator: (v: T) => K): any;
```

Comment:
In order to avoid `any`, you could use overloads based on the `returns` option value.